modules/master-node-sizing.adoc

```diff
@@ -6,21 +6,36 @@
 [id="master-node-sizing_{context}"]
 =  Control plane node sizing
 
-The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing.
-
-[options="header",cols="3*"]
+The control plane node resource requirements depend on the number of nodes in the cluster. The following control plane node size recommendations are based on the results of control plane density focused testing. The control plane tests create the following objects across the cluster in each of the namespaces depending on the node counts:
+
+----
+12 imagestreams
```

Comment:
I would make this a bulleted list and take it out of the code block

Reply:
Sure. Fixed it, thanks.